metaproc-connector-demo/config/variables.yaml

```diff
@@ -3,3 +3,4 @@ Variables:
     Url: http://localhost:8080
     Username: admin
     Password: 1234
+    TokenExpiredDuration: 3600000 # Milisecond
```

Comment:
this is an invalid and broken yaml: comments should reside on top of the variable name
https://dev.axonivy.com/doc/10.0/designer-guide/configuration/variables.html#variables

Reply:
Also I think using milliseconds here is not what the customer wants,
A duration of seconds or better minutes is much easier to setup

Reply:
Hi @ivy-rew, so the variable should start with lower case right?

Reply:
No, as you see there are other variables starting with capital letters. So I'd keep this pattern.